src/methods/democracy/types.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * A vote in a referendum
+ */
+export type Vote = {
```

Comment:
Since this Vote is really specific to proxyVote, I'd rather have this definition inside proxyVote.ts

Reply:
`Vote` is used in vote.ts and proxyVote.ts

Reply:
Ah, right! Then everything's good